### DISCUSSION TITLE
How to add and remove interfaces within Redistribution Profile

### DISCUSSION THREAD

**Discussion by bfreebu** [PaloAltoNetworks/pan-os-python#349] [Q&A]
Hello,

I'm new to pan-os-python and like to know what do I need to do in order to add/remove interfaces within the redistribution profile.  

Thank you!

**Comment by shinmog** [comment]
You're looking for the `.create()` / `.apply()` / `.delete()` methods, they exist on all objects.

If you look in the "examples" directory there is a smattering of heavily documented scripts that goes over the basics.
